docs/source/run/service.rst

```diff
@@ -0,0 +1,93 @@
+====================
+Running as a service
+====================
+
+Now that you can run the bot, you probably want it to run forever in the
+background of your system. While there are many ways to do so, there are a few
+steps and requirements that remain the same to properly run Sopel as a service.
+
+
+Requirements to run a service
+=============================
+
+Sopel needs the following:
+
+* a non-root user to run as
+* read/write access to its configuration file
+* a writable pid directory
+  (:attr:`~sopel.config.core_section.CoreSection.pid_dir`)
+* a writable logs directory
+  (:attr:`~sopel.config.core_section.CoreSection.logdir`)
+* a writable data directory
+  (:attr:`~sopel.config.core_section.CoreSection.homedir`)
+
+.. important::
+
+    Do **not** run Sopel as the **root** user. And avoid if possible to run
```

Comment:
If this isn't already the case, sopel should probably just check this and quit. Better to enforce with code than docs.

Reply:
Without going into that debate: the code already does that, but this warning is less for the regular user, and more for anyone tempted to do something rather stupid. It is a warning to remind the reader that, hey, be careful out there, it's dangerous.